spinalcordtoolbox/gui/centerline.py

```diff
@@ -127,8 +130,8 @@ def mode(self, value):
 
         if value != self._mode:
             self._mode = value
-            self.points = []
-            self.reset_position()
+            # self.points = []
+            # self.reset_position()
```

Comment:
Yes, I think this can be safely removed, thank you.
done in https://github.com/neuropoly/spinalcordtoolbox/pull/3175/commits/dbbde66418bb93a112d0306eedce4eec395e2a1e